src/contexts/states/index.ts

```diff
@@ -7,18 +7,16 @@ import initComponents from './components'
 
 export type FetchStatus = keyof State.FetchStatus
 
-const initHomeBlocks = [] as State.Block[]
-const initStatisticsChartDatas = [] as State.StatisticsChartData[]
-
 const initState: State.AppState = {
   app: initApp,
   blockState: initBlockState,
   blockListState: initBlockListState,
   addressState: initAddressState,
   transactionState: initTransactionState,
   statistics: initStatistics,
-  statisticsChartDatas: initStatisticsChartDatas,
-  homeBlocks: initHomeBlocks,
+  statisticsChartDatas: [] as State.StatisticsChartData[],
```

Comment:
There's no need to assert the type `as State.StatisticsChartData` if it's declared in the interface `State.AppState`

Reply:
OK

Reply:
Updated. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/353/commits/3896cdba4af22bc1401f3678fb17ad27e153c367